### PR TITLE
ERMS-3936-3

### PR DESCRIPTION
**Pull request by Inselhopper**

### CODE REVIEW

File: grails-app/i18n/messages.properties [hunk 1]
```diff
@@ -3044,6 +3044,12 @@ myinst.dash.due_dates.attribute.label = Attribute
...
 myinst.dash.due_dates.done.label = Done
 myinst.dash.due_dates.hide.label = Hide
+myinst.dash.due_dates.visibility.label = Visibility
+myinst.dash.due_dates.status.label = Status
+myinst.dash.due_dates.visibility.on.tooltip =  Turn on visibility
+myinst.dash.due_dates.visibility.off.tooltip = Turn off visibility
+myinst.dash.due_dates.status.done.tooltip = Set done
+myinst.dash.due_dates.status.pending.tooltip = Set pending
 
 myinst.addLicense.label = Data import explorer
```
Comment (croqueGrec09) [inline grails-app/i18n/messages.properties:3054]: English text for myinst.dash.due_dates.attribute.label?

File: grails-app/views/user/_tableDueDates.gsp [hunk 2]
```diff
@@ -146,12 +148,13 @@
...
                 <g:if test="${dashDueDate?.dueDateObject.isDone}">
-                    <laser:remoteLink class="ui green button la-modern-button"
+                    <laser:remoteLink class="ui green button la-modern-button la-popup-tooltip la-delay"
+                                      data-content="${message(code:'myinst.dash.due_dates.status.pending.tooltip')}"
                                       controller="ajax"
                                       action="dashboardDueDateSetIsUndone"
                                       params='[owner:"${dashDueDate.dueDateObject.class.name}:${dashDueDate.dueDateObject.id}"]'
                                       id="${dashDueDate.dueDateObject.id}"
-                                      data-confirm-tokenMsg="Möchten Sie diesen fälligen Termin auf NICHT erledigt sezten? "
+                                      data-confirm-tokenMsg="Möchten Sie diesen fälligen Termn auf NICHT erledigt sezten? "
```
Comment (croqueGrec09) [inline grails-app/views/user/_tableDueDates.gsp:157]: typo: Termin

**Review by croqueGrec09 — COMMENTED** [review]
remarks

File: grails-app/i18n/messages.properties (hunk 1 above)
Comment (Inselhopper) [reply grails-app/i18n/messages.properties:3054]: myinst.dash.due_dates.attribute.label = Attribute
Zeile 3043 war schon da

**Pull request merged**